src/ValueFormatters/QuantityFormatter.php

```diff
@@ -178,23 +219,25 @@ protected function formatNumber( QuantityValue $quantity ) {
 	}
 
 	/**
-	 * Returns the rounding exponent based on the given $quantity
-	 * and the @see QuantityFormatter::OPT_APPLY_ROUNDING option.
+	 * Returns the rounding exponent based on the given $quantity.
 	 *
-	 * @param QuantityValue $quantity
+	 * @param UnboundedQuantityValue $quantity
+	 * @param bool|int $roundingMode The rounding exponent, or true for rounding to the order of
+	 *        uncertainty (significant digits) of a QuantityValue, or false to not apply
+	 *        rounding (that is, round to all digits).
 	 *
 	 * @return int|null
 	 */
-	private function getRoundingExponent( QuantityValue $quantity ) {
-		if ( $this->options->getOption( self::OPT_APPLY_ROUNDING ) === true ) {
-			return $this->options->getOption( self::OPT_SHOW_UNCERTAINTY_MARGIN )
-				? null
-				// round to the order of uncertainty
-				: $quantity->getOrderOfUncertainty();
-		} elseif ( $this->options->getOption( self::OPT_APPLY_ROUNDING ) === false ) {
-			return null;
+	private function getRoundingExponent( UnboundedQuantityValue $quantity, $roundingMode ) {
+		if ( is_int( $roundingMode ) ) {
+			// round to the given exponent
+			return $roundingMode;
+		} elseif ( $roundingMode && ( $quantity instanceof  QuantityValue ) ) {
+			// round to the order of uncertainty (QuantityValue only)
+			return $quantity->getOrderOfUncertainty();
 		} else {
-			return (int)$this->options->getOption( self::OPT_APPLY_ROUNDING );
```

Comment:
This looks like an unrelated breaking change. Before the code accepted numeric strings. Now it seems it ignores the option if e.g. `"2"` is provided as a string.